### PR TITLE
Feature/template rework test

### PR DESCRIPTION
**Pull request by KiriHoy**
A draft re-work of the current templates page

### CODE REVIEW

File: .templates/new-module/code/index.yml [hunk 1]
```diff
@@ -12,9 +12,8 @@ sidebar:
   - /components/sidebar.md
 
 main:
-  - react-usage.md
-  - react-props.md
-  - react-exports.md
```
Comment (sukhrajghuman) [inline .templates/new-module/code/index.yml]: any reason why we made changes to this file?

File: src/layout/templates/demo.js [hunk 2]
```diff
@@ -31,12 +31,7 @@ const TemplateDemo = ({ _ID, _body, _relativeURL }) => {
 				<div className="col-sm-6">
 					{ _body }
 					<ul className="au-btn__list au-btn__list--inline">
-						<li><AUbutton link="https://github.com/govau/design-system-starter/archive/master.zip">
-							Download
-						</AUbutton></li>
-						<li><AUbutton link="/get-started/starter-kit" as='secondary'>
-							Get started
-						</AUbutton></li>
+						<li><AUbutton link={ `${ templateID }/customise` } as='primary'>Customise</AUbutton></li>
```
Comment (sukhrajghuman) [inline src/layout/templates/demo.js]: if it's only one button, does it need to be in a list?

File: src/sass/components/_templates.scss [hunk 3]
```diff
@@ -63,11 +62,11 @@
...
 
-	@include AU-media( xs ){
-		@include AU-space( margin-left, 19unit );
-		margin-top: 0;
-		float: right;
-	}
+	// @include AU-media( xs ){
+	// 	@include AU-space( margin-left, 19unit );
+	// 	margin-top: 0;
+
```
Comment (sukhrajghuman) [inline src/sass/components/_templates.scss]: remove these

File: src/sass/components/_templates.scss [hunk 4]
```diff
@@ -49,10 +49,9 @@
 
 
 // The image inside the card
-.template-card .template-card__image {
-	@include AU-space( max-width, 24unit );
+.template-card__image {
 	@include AU-space( max-height, 18unit );
-	@include AU-space( margin-top, 2unit );
+	//@include AU-space( margin-top, 2unit );
```
Comment (sukhrajghuman) [inline src/sass/components/_templates.scss]: remove if not needed

**Comment by KiriHoy** [comment]
> can we also add back the `.templates` folder
That should be back in now

**Pull request merged**